### PR TITLE
Feature/pt wear record notifications

### PR DESCRIPTION
**Pull request by finger563**

### CODE REVIEW

**Comment by finger563** [comment]
@bradmartin  the push notifications seem to be good (logic-wise) and i've tested the push notification for coast time. relying on multiple days data will take a little longer to test for the push warning, but i'll continue that even after we merge this.

For now - lets focus on cleaning up the 
* notification channels (so we have the right ones and they're properly localised / have proper descriptions)
* notification titles / text (localization)

Once we get that cleaned up I think we'll probably be good to merge.

File: apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java [hunk 1]
```diff
@@ -798,6 +969,63 @@ private void sendMessageToActivity(String msg, String extraKey) {
...
+                .setContentTitle(getString(R.string.new_coast_time_record))
+                .setContentText(getString(R.string.coast_time_record_notification_text))
+                .setColor(0x006ea5)
+                .setSmallIcon(R.drawable.ic_notification_icon)
+                .setLargeIcon(Icon.createWithResource(this, R.drawable.ic_notification_icon))
+                .setChannelId(R.string.personal_record);
+
+        NotificationManager notificationManager =
+                (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        notificationManager.notify(20001, notificationBuilder.build());
```
Comment (finger563) [inline apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java:1026]: @bradmartin don't these have to be different numbers? - e.g. they can't both use `20001` in the `notify(...)` call?

Comment (bradmartin) [reply apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java:1026]: ```passing it a unique ID for the notification and the result ``` but it's really only necessary to KNOW the ID if we plan to interact on it later in some way (cancel it or handle user interaction). If we're not doing anything it's prob fine, or just append some random `int` on the end of it from the current system clock (whatever is easiest)

Comment (finger563) [reply apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java:1026]: Yea I guess I don't fully understand their documentation - on the next overload, they say that the `tag, id` must be unique within our app, but they *don't* say for this call that the `id` must be unique within our app so I'm not really sure.

File: apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java [hunk 2]
```diff
@@ -798,6 +969,63 @@ private void sendMessageToActivity(String msg, String extraKey) {
...
+
+    @RequiresApi(api = Build.VERSION_CODES.O)
+    private void showCoastTimeRecordNotification() {
+        Builder notificationBuilder = new Builder(this, Constants.NOTIFICATION_CHANNEL)
+                .setContentTitle(getString(R.string.new_coast_time_record))
+                .setContentText(getString(R.string.coast_time_record_notification_text))
+                .setColor(0x006ea5)
+                .setSmallIcon(R.drawable.ic_notification_icon)
+                .setLargeIcon(Icon.createWithResource(this, R.drawable.ic_notification_icon))
+                .setChannelId(R.string.personal_record);
```
Comment (finger563) [inline apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java:1022]: @bradmartin I'm getting `.setChannelId(R.string.personal_record)` (and `smartdrive`) both giving the error that `setChannelId(...)` require an int, not a string

Comment (finger563) [reply apps/wear/pushtracker/App_Resources/Android/src/main/java/com/permobil/pushtracker/wearos/ActivityService.java:1022]: actually it was the other way around `R.string.personal_record` actually returns an `int`, so we just have to wrap them in `getString(...)`

**Review by bradmartin — APPROVED** [review]

**Pull request merged**